### ISSUE TITLE
Organization Plan description error confusion

### ISSUE THREAD

**Issue by eric29** [bitwarden/server#520]
Description for the "Team" plan is contradicting itself, and not sure which one is correct.

Teams:
For businesses and other team organizations.
• Add and share with **unlimited users**
• Create unlimited collections
• 1 GB encrypted file storage
• Priority customer support
• 7 day free trial, cancel anytime
$5.00 /month, **includes 5 users**, additional users $2.00 /month

Or is this a misleading marketing tactic for "it's unlimited if you pay for each user"?

**Comment by kspearrin** [comment]
There is no limit on the number of users you can add, like the Families plan, which has a max of 5 users. The teams plan comes with 5 base user seats. You can add unlimited more for $2 per user per month.

**Comment by eric29** [comment]
@kspearrin

Alright, thanks. They should rephrase it to better understand. Cause some people would have paid for this only to be confused and upset that they still have to pay for users on top of paying more.

**Comment by djsmith85** [comment]
The [pricing page](https://bitwarden.com/pricing/business/) has been re-worked since then. The price per user/month can be seen at the bottom of the comparison table, and also when hovering over the i next to Teams it displays the following message: `The Teams plan is priced per user. You can add or remove users at anytime.`

Closing this now.
